akka-http-core/src/main/resources/reference.conf

```diff
@@ -206,6 +206,25 @@ akka.http {
       # be increased for high bandwidth-delay-product connections.
       incoming-stream-level-buffer-size = 512kB
     }
+
+    websocket {
+      # periodic keep alive may be implemented using by sending Ping frames
+      # upon which the other side is expected to reply with a Pong frame,
+      # or by sending a Pong frame, which serves as unidirectional heartbeat.
+      # Valid values:
+      #   ping - default, for bi-directional ping/pong keep-alive heartbeating
+      #   pong - for uni-directional pong keep-alive heartbeating
+      #
+      # It is also possible to provide a payload for each heartbeat message,
+      # this setting can be configured programatically by modifying the websocket settings.
+      # See: https://doc.akka.io/docs/akka-http/current/server-side/websocket-support.html
+      periodic-keep-alive-mode = ping
+
+      # Interval for sending periodic keep-alives
+      # The frame sent will be the onne configured in akka.http.server.websocket.periodic-keep-alive-mode
+      # `infinite` by default, or a duration that is the max idle interval after which an keep-alive frame should be sent
```

Comment:
does "infinite" mean that no ping-pong are sent? perhaps spell that out in the description?

Reply:
Yes, that's what it means -- will explain it more then